src/python/pants/ivy/bootstrapper.py

```diff
@@ -157,7 +158,8 @@ def _bootstrap_ivy(self, bootstrap_jar_path):
         except fetcher.Error as e:
           raise self.Error('Problem fetching the ivy bootstrap jar! {}'.format(e))
 
+    resolution_cache_dir = options.resolution_cache_dir if options.resolution_cache_dir else options.cache_dir
```

Comment:
The `None` check seems to be repeated a few places... would be good to use the `IvySubsystem` method to get this value.